test/integration/features/F002_Invalid_Change_Events.feature

```diff
@@ -158,15 +158,26 @@ Feature: F002. Invalid change event Exception handling
     Then the Event Processor logs to splunk with report key "INVALID_OPEN_TIMES"
 
 @complete @cloudwatch_queries
-  Scenario: F002S021. Dentist Unmatched Service Type uses correct report key
+  Scenario Outline: F002S021. Dentist Unmatched Pharmacy and Service report keys
     Given a Dentist Changed Event is valid
-    And the Changed Event has ODS Code "FQG8101"
+    And the Changed Event has ODS Code "<ods_code>"
     When the Changed Event is sent for processing with "valid" api key
-    Then the Event Processor logs to splunk with report key "UNMATCHED_SERVICE_TYPE"
```

Comment:
We are not actually checking its logged to splunk so maybe we should change the name of this to "... logs with report key ...."